type_wrapper.py

```diff
@@ -35,3 +35,11 @@ class NoValue(Value):
     def __init__(self, value=None):
         self.value = value
 
+class BasicBlock(Value):
+    def __init__(self, value=None):
```

Comment:
Uh, I need to remove the default value... They're all wrappers so that I can return various values from the method that executes an operation.